tests/functional/restart/34-auto-restart-basic.t

```diff
@@ -44,6 +44,9 @@ init_suite "${TEST_NAME}" - <<'__FLOW_CONFIG__'
 [scheduling]
     [[graph]]
         R1 = "task<foo> => task<foo+1>"
+[runtime]
+    [[task<foo>]]
+    [[task_26]]
```

Comment:
As long as this works. (I would have gone for `[scheduler]allow implcit tasks = True`)

Reply:
I'm going to let it stand, although either would be fine, I don't think it's better enough to do stuff about.